packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -389,7 +389,7 @@ async function createFile(entry, options, method, fileId) {
   }
   if (options.contentType) {
     if (options.contentType === true && entry.filename) {
-      createFileOptions.contentType = mimetypes.contentType(entry.filename)
+      createFileOptions.contentType = mimetypes.contentType(path.extname(entry.filename))
```

Comment:
@doubleface returned content-type should probably be checked against `false` and replaced with `application/octet-stream` in that case

```
> mime.lookup(extname("file_with_no_extension"))
false
> mime.lookup("file_with_no_extension")
false
```